trompace/client/client_itemlist.py

```diff
@@ -0,0 +1,128 @@
+
+from trompace.config import config
+from trompace.connection import submit_query
+from trompace.mutations.itemlist import *
```

Comment:
it's normally recommended to not use `import *`, to make it more explicit as to where your methods are coming from. In this case I'd do:
`from trompace.mutations import itemlist as mutations_itemlist` and then when you need to call a method: `mutations_itemlist.mutation_create_itemlist(...)`